tests/Giraffe.Tests/Giraffe.Tests.fsproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net7.0;net8.0</TargetFrameworks>
+    <TargetFrameworks>net6.0;net7.0;net8.0;net9.0</TargetFrameworks>
```

Comment:
Do you aim to remove net6 and net7 in a different PR?
Feels weird to keep them.

Reply:
Yeah, I think it makes sense to remove net6/7 from testing.

I really want Microsoft to have clear documentation on when it makes sense to update a TFM. Because I don't think there's a super compelling reason to build Giraffe project for 9 if we're not doing any ifdefs.

Reply:
I get this warning when building this project locally:

```
ResponseCachingApp succeeded with 1 warning(s) (2.4s) → samples/ResponseCachingApp/bin/Debug/net7.0/ResponseCachingApp.dll
    /.../sdk/9.0.101/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.EolTargetFrameworks.targets(32,5): warning NETSDK1138: The target framework 'net7.0' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/dotnet-core-support for more information about the support policy.
```

Reply:
Removed with this commit https://github.com/giraffe-fsharp/Giraffe/pull/639/commits/0cfba09cd534fe97dcd9e075a671735dc18cf8c0